Resources/public/js/views/services/ez-locationviewviewservice.js

```diff
@@ -186,6 +226,50 @@ YUI.add('ez-locationviewviewservice', function (Y) {
         },
 
         /**
+         * Delete content item callback triggering notifications and making app to navigate to parent location
+         *
+         * @method _afterDeleteCallback
+         * @protected
+         * @param {eZ.Location} parentLocation the parent location to which app will navigate to
+         * @param {eZ.Content} content the content item that has been deleted
+         * @param {Boolean} error
+         */
+        _afterDeleteCallback: function (parentLocation, content, error) {
+            var app = this.get('app'),
+                contentName = content.get('name');
+
+            if (error) {
+                this._notify(
+                    'An error occurred when deleting "' + contentName + '"',
+                    'content-' + content.get('id'),
```

Comment:
you're right @StephaneDiot. My fault. i'll add these and blank lines fixes to last changes proposed by @dpobel. 
For the icon part, yep, @dpobel suggested me to use the same icon as the send to trash for now. 
Thanks for your input!.